### ISSUE TITLE
Support a mix of stable and unstable versions

### ISSUE THREAD

**Issue by Raicuparta** [filllabs/dependi#126]
**Is your feature request related to a problem? Please describe.**
It's common to need unstable versions of specific dependencies, while wanting all other dependencies to use the stable versions. Currently the solution is to set dependi to not exclude unstable versions, and then manually pick the latest stable for each individual dependency, which is a very slow process. The latest stable is often lost in a long list of in-between unstables, and of course if you have a lot of dependencies this is very annoying. It's often faster to just not use dependi at all and go with the command line instead.

**Describe the solution you'd like**
Instead of having only an "exclude unstable" checkbox, we could turn that option into an enum where you can pick what to do with unstable versions:
  1. Exclude them (default)
  2. Include them and always consider them as latest (if they're more recent than the stable versions of course).
  3. Include them, but only if the current version is already unstable.

Node example when using option 3:

```jsonc
// package.json

// Let's use prettier as an example, which currently has:
// latest stable: 3.3.3
// latest unstable: 4.0.0-alpha.10

"dependencies": {
  // Considered up to date because it's the latest unstable,
  // and 4.0.0-alpha.10 is unstable, so we include unstable versions.
  "prettier": "4.0.0-alpha.10",  ✅

  // Considered outdated because there's a later unstable,
  // and 4.0.0-alpha.4 is unstable, so we include unstable versions.
  "prettier": "4.0.0-alpha.4",  ❌ 4.0.0-alpha.10

  // Considered up to date even though the latest unstable is 4.0.0-alpha,
  // because 3.3.3 is stable, so we exclude unstable versions.
  "prettier": "3.3.3", ✅

  // Considered outdated because there's a later stable,
  // and 3.0.0 is stable, so we exclude unstable versions.
  "prettier": "3.0.0",  ❌ 3.3.2
}
```

**Describe alternatives you've considered**
Another thing that could help with this, although still far from ideal, would be somehow separating the unstable and stable versions from the list that shows up on the hover tooltip. This would at least make it easier to find a stable version for packages that have the stables lost within hundreds of automatically-deployed unstable versions. Or maybe before listing the entire dependency list, just have two separate items for "latest stable" and "latest unstable".

**Comment by Raicuparta** [comment]
Looks like this was added in 0.7.10 even though it was added to the 0.7.11 milestone. Seems to work exactly as expected.

**Comment by MOzanGul** [comment]
Hi @Raicuparta  , sorry for the confusion. This issue was developed faster than our plans thanks to our great developers. 
We have fixed this issue per your request in 0.7.10.
We'd be really happy if you could review Dependi, here is our [reviews page](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi&ssr=false#review-details).
